core/static/js/dit.class.modal.js

```diff
@@ -132,14 +137,28 @@
       }
     });
 
+    self.$firstLink.on("keydown", function(e) {
+      if (e.shiftKey && e.which === 9) {
+      // Loop around to close button when pressing
+      // shift+tab on first link
+        e.preventDefault();
+        self.$closeButton.focus();
+      }
+    });
+
     self.$closeButton.on("click keydown", function(e) {
       // Close on click or Enter
       if(e.which === 1 || e.which === 13) {
         Modal.deactivate.call(self);
         e.preventDefault();
+      } else if (e.which === 9 && !e.shiftKey) {
+      // After pressing tab when close button is selected
+      // loop back around to the first link
+        self.$firstLink.focus();
```

Comment:
This should just call self.focus(), since that should refocus to the required first element.